cypress/e2e/signIn.cy.js

```diff
@@ -12,23 +17,40 @@ describe('Sign In page', () => {
 
   before(() => {
     cy.task('db:clear');
+  });
+
+  beforeEach(() => {
     cy.task('generateUser').then((generateUser) => {
```

Comment:
The variable 'user' is being used before it is defined. This will cause a runtime error if 'user' is not in the global scope. You should declare 'user' before using it.

Reply:
I declared 'user' variable at row 16:

describe('Sign In page', () => {
  let user;

  before(() => {